src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -408,6 +410,8 @@ public void RemovePeer(PublicKey nodeId)
 
         public AcceptTxResult SubmitTx(Transaction tx, TxHandlingOptions handlingOptions)
         {
+            if (_headInfo.IsSyncing) return AcceptTxResult.AlreadyKnown;
```

Comment:
Shouldn't we have a new result for this scenario?

Reply:
+1 for creating `AcceptTxResult.Syncing` or something like that

Reply:
Added